src/impl_/exceptions.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{
+    sync::GILOnceCell,
+    types::{PyAnyMethods, PyTracebackMethods, PyType},
+    Bound, Py, Python,
+};
+
+pub struct ImportedExceptionTypeObject {
+    imported_value: GILOnceCell<Py<PyType>>,
+    module: &'static str,
+    name: &'static str,
+}
+
+impl ImportedExceptionTypeObject {
+    pub const fn new(module: &'static str, name: &'static str) -> Self {
+        Self {
+            imported_value: GILOnceCell::new(),
+            module,
+            name,
+        }
+    }
+
+    pub fn get<'py>(&self, py: Python<'py>) -> &Bound<'py, PyType> {
```

Comment:
Isn't this basically the same as `GILOnceCell<Py<PyType>>::get_or_try_init_type_ref`?

Reply:
Yes, I'll deduplicate and push a changelog entry when I get a chance later.